### PR TITLE
Pull Arrow nightlies from arrow-nightlies

### PR DESCRIPTION
**Pull request by xhochy**
Permanently moved.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ install:
   - conda activate test-env
   - >
     if [[ "${ARROW_NIGHTLY-0}" == 1 ]]; then
-        travis_retry conda install -c ursa-labs/label/nightly pyarrow
+        travis_retry conda install -c arrow-nightlies arrow-cpp pyarrow
```
Comment (lr4d) [inline .travis.yml]: do we need `arrow-cpp`?

Comment (xhochy) [reply .travis.yml]: Locally I had the case that only `pyarrow` was pulled from the repo but `arrow-cpp` stayed on the conda-forge version. The two must though exactly match in the version used, so both should be pulled. The correct behaviour would be that `arrow-cpp` as a dependency would be automatically also be pulled from the repo but this is not yet happening consistently.

Comment (lr4d) [reply .travis.yml]: maybe add that as a comment for posterity

**Comment by xhochy** [comment]
@lr4d @fjetter Merge?

**Comment by lr4d** [comment]
Thanks @xhochy

**Pull request merged**